dev-docs/modules/browsiRtdProvider.md

```diff
@@ -55,7 +55,9 @@ Syntax details:
 {: .table .table-bordered .table-striped }
 | Name  |Type | Description   | Notes  |
 | :------------ | :------------ | :------------ |:------------ |
-| name  | String | Real time data module name | Always 'browsi' | 
+| name  | String | Real time data module name | Always 'browsi' |
+| auctionDelay  | Number | Max time in ms to delay the auction | Optional. Default to 0 |
```

Comment:
I think `auctionDelay` should be up at the higher level underneath realTimeData, not within the dataProviders section. 

```
    pbjs.setConfig({
        realTimeData: {
            auctionDelay: 100,    // global to all sub-modules
            dataProviders:[{          
                name: "SUB-MODULE-NAME",
                params: {
                    // sub-module specific parameters
                }
            }]
        }
    });
```

Reply:
Has this question been resolved? This PR is very old and the accompanying PBJS PR has been merged.